image_generation/lcm_dreamshaper_v7/cpp/requirements.txt

```diff
@@ -0,0 +1,4 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+torch==2.2.2+cpu
+diffusers==0.27.2
+optimum-intel[nncf,openvino] @ git+https://github.com/apaniukov/optimum-intel.git@0029e9165a2dad4cfcf787aa63181d9dc0cd49d5
```

Comment:
`nncf` is not required, because `openvino` adds it https://github.com/apaniukov/optimum-intel/blob/main/setup.py#L63